(AzureForums) Rename the section Deploy the Azure Stack Development Kit

AjayKumar-N opened the pull request:
Kindly rename the section "Deploy Azure Stack Development Kit" to "Azure Stack deployment prerequisites" under the "Next steps" section.

azurecla commented:
Hi __@yossibanai__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

BryanLa commented:
Working with @yossibanai and @curtand to get the changes redirected to the private repo.

BryanLa commented:
Closing for now, as there have been several updates to article since submitting.  We will continue to work with Yossi (and the entire AAD engineering team) to redirect changes via private repo.

AjayKumar-N committed:
(AzureForums) Rename the section Deploy the Azure Stack Development Kit

Kindly rename the section "Deploy Azure Stack Development Kit" to "Azure Stack deployment prerequisites" under the "Next steps" section.
